18_links_lab/linkslab.txt

```diff
@@ -2,19 +2,32 @@ Preparation: Perform the following  (no need to write the commands you used):
 
   - create a new directory named "links"
   - cd to it
-  - inside the directory create 2 text files named "file1" and "file2". Write the names of some things you like in file "file1", and some things you don't like in file "file2"
+  - inside the directory create 2 text files named "file1" and "file2". 
+Write the names of some things you like in file "file1", and some things you don't like in file "file2"
 
+
 1. Perform the following and write the commands you used, from within links directory:
 
   - Create a symbolic link named file3 that points to "file1"
-  Answer: 
+  
+Answer: 
 
-  - Create a hard link named file4 that points to "file2"
-  Answer: 
+ ln -s file1 file3
+- Create a hard link named file4 that points to "file2"
+  
+Answer: 
 
-  - Create a new directory named "tmp" in your home folder:
-  Answer:
+  ln file2 file4
+- Create a new directory named "tmp" in your home folder:
+  
+Answer:
 
-2. Move "file3" to the new "~/tmp" directory you just created. Can you still use it to see the original "file1" ? Why not? How would you create a symbolic link that would survive the transfer to ~/tmp ?
+   mkdir /home/avia/tmp

```

Comment:
Correct, and also worth knowing about ~ which is a shorthand to home directory, so this one works too: `mkdir ~/tmp`